app/controllers/api/v1/authentication_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class AuthenticationController < Api::V1::ApiController
+      skip_before_action :authenticate_request
+
+      def login
+        @user = User.find_by_email(params[:email])
```

Comment:
Also, try to avoid the `find_by_(attribute)` way. It's convention to use `find_by(attribute: )`